ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/AddRemoveEditPanel.java

```diff
@@ -0,0 +1,34 @@
+package com.drewhannay.chesscrafter.panel;
+
+import javax.swing.Box;
+import javax.swing.BoxLayout;
+import javax.swing.DefaultListModel;
+import javax.swing.JButton;
+import javax.swing.JPanel;
+import javax.swing.ListCellRenderer;
+import java.awt.Dimension;
+
+public class AddRemoveEditPanel extends JPanel {
```

Comment:
This should extend `ChessPanel` and call `super(false)` so it won't paint the purple background